src/templates/BlogPost.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react"
+import { Helmet } from "react-helmet"
+import Layout from '../components/layout'
+
+import './BlogPost.css'; // make it pretty!
+
+export default function Template({
+  data, // this prop will be injected by the GraphQL query we'll write in a bit
+}) {
+  const { markdownRemark: post } = data // data.markdownRemark holds our post data
+  return (
+		<Layout noHero>
+			<Helmet title={`OSS by Zenika - ${post.frontmatter.title}`} />
+			<div className="blog-post">
```

Comment:
We should have a better header for the article (with the title and maybe the summary)

Reply:
And also add a bigger margin to the page
_(You should show it quickly to Stéphane and update it with him)_